external_samples/component.py

```diff
@@ -0,0 +1,68 @@
+from abc import ABC, abstractmethod
```

Comment:
ABC isn't used

Reply:
oops.   It should have been part of component.  Good catch